app/Platform/Services/DeveloperFeedService.php

```diff
@@ -106,7 +106,7 @@ private function attachRecentAwardRowsMetadata(mixed $awardRows): mixed
             $row->GameTitle = $game->Title ?? null;
             $row->GameIcon = $game->ImageIcon ?? null;
 
-            $row->Untracked = $userData[$row->User]->Untracked ?? null;
+            $row->Untracked = $userData[$row->user->User]->Untracked ?? null;
```

Comment:
You should be able to eliminate the `$userData` query on line 92. On line 91 just `loadMissing('user')` and change this to `$row->user->Untracked ?? null`.

Reply:
Fixed in latest